src/main/java/com/denizenscript/depenizen/bukkit/objects/superiorskyblock/SuperiorSkyblockIslandTag.java

```diff
@@ -0,0 +1,397 @@
+package com.denizenscript.depenizen.bukkit.objects.superiorskyblock;
+
+import com.bgsoftware.superiorskyblock.api.SuperiorSkyblockAPI;
+import com.bgsoftware.superiorskyblock.api.enums.Rating;
+import com.bgsoftware.superiorskyblock.api.island.Island;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.Adjustable;
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+
+public class SuperiorSkyblockIslandTag implements ObjectTag, Adjustable {
+
+    // <--[ObjectType]
+    // @name SuperiorSkyblockIslandTag
+    // @prefix superiorskyblock_island
+    // @base ElementTag
+    // @format
+    // The identity format for group is <island_name>
+    // For example, 'superiorskyblock_island@denizen_scripters'.
+    //
+    // @plugin Depenizen, SuperiorSkyblock
+    // @description
+    // A SuperiorSkyblockIslandTag represents a SuperiorSkyblock island.
+    //
+    // -->
+
+    @Fetchable("superiorskyblock_island")
+    public static SuperiorSkyblockIslandTag valueOf(String string, TagContext context) {
+        if (string.startsWith("superiorskyblock_island@")) {
+            string = string.substring("superiorskyblock_island@".length());
+        }
+        Island island = SuperiorSkyblockAPI.getIsland(string);
```

Comment:
Islands seem to have both UUIDs and names, probably better to have the objects identify based on the UUIDs rather than names, as the names are changeable.